core/node/api_server/src/execution_sandbox/validate.rs

```diff
@@ -39,13 +39,16 @@ impl SandboxExecutor {
         block_args: BlockArgs,
         fee_input: BatchFeeInput,
         whitelisted_tokens_for_aa: &[Address],
-    ) -> Result<(), ValidationError> {
+    ) -> Result<ValidationTraces, ValidationError> {
         let total_latency = SANDBOX_METRICS.sandbox[&SandboxStage::ValidateInSandbox].start();
         let validation_params = get_validation_params(
             &mut connection,
             &tx,
             self.options.eth_call.validation_computational_gas_limit(),
             whitelisted_tokens_for_aa,
+            self.timestamp_asserter_address,
+            self.timestamp_asserter_min_range_sec,
+            self.timestamp_asserter_min_time_till_end_sec,
```

Comment:
IIUC, during node synchronization it can still accept transaction requests (i.e., validate transactions). If this is the case, is the fact that validation can be called with `timestamp_asserter_address == Some(_)` before the asserter contract is deployed a concern?